app/dashboard/page.tsx

```diff
@@ -1,7 +1,33 @@
-import React from "react";
+import Card from "@/components/dashboard/Card/Card";
+import CarouselWrapper from "@/components/dashboard/Carousel/CarouselWrapper";
 
 const Dashboard = () => {
-	return <div>Hello from dashboard!</div>;
+	return (
+		<div>
+			<header></header>
+			<main>
+				<CarouselWrapper>
+					<Card isTaken={false} hour="08:00" name="Ketonal" dosage="120mg" />
+					<Card isTaken={false} hour="10:00" name="Apap" dosage="30mg" />
+					<Card isTaken={false} hour="12:00" name="Antibiotic" dosage="5mg" />
+					<Card
+						isTaken={false}
+						hour="16:00"
+						name="Paracetamol"
+						dosage="120mg"
+					/>
+					<Card isTaken={false} hour="22:00" name="Ibuprom" dosage="3mg" />
+					<Card isTaken={false} hour="22:00" name="Ibuprom" dosage="3mg" />
+					<Card isTaken={false} hour="22:00" name="Ibuprom" dosage="3mg" />
+					<Card isTaken={false} hour="22:00" name="Ibuprom" dosage="3mg" />
+					<Card isTaken={false} hour="22:00" name="Ibuprom" dosage="3mg" />
+					<Card isTaken={false} hour="22:00" name="Ibuprom" dosage="3mg" />
+					<Card isTaken={false} hour="22:00" name="Ibuprom" dosage="3mg" />
+					<Card isTaken={false} hour="22:00" name="Ibuprom" dosage="3mg" />
+				</CarouselWrapper>
```

Comment:
jak rozumiem, to tylko mock? pobieranie danych będzie później, si?

jak widzisz tutaj masz wszędzie `isTaken={false}` i właściwie defaltowa wartość powinna być false, przez co nie musiałabyś się tak powtarzać

Reply:
Tak, z tego, co pamiętam, to tak się umówiłyśmy. Dałam więcej żeby zobaczyć jak zadziała karuzela